src/index.ts

```diff
@@ -104,12 +114,126 @@ class ComputerSpeakers implements Accessory {
           this.getVolume.bind(this, logarithmic)
         )
     }
+
+    if (!!delta) {
+      log.debug(
+        "Creating stateless switches for increasing and decreasing volume"
+      )
+
+      this.increaseVolumeSwitch = new Service.Switch(
+        name + " +" + delta + "%",
+        ConfigService.IncreaseVolumeButton
+      )
+
+      this.increaseVolumeSwitch
+        .getCharacteristic(Characteristic.On)
+        .on(
+          CharacteristicEventTypes.SET,
+          this.adjustVolume.bind(this, logarithmic, delta, delay)
+        )
+        .on(CharacteristicEventTypes.GET, this.showAlwaysOff.bind(this))
+
+      this.decreaseVolumeSwitch = new Service.Switch(
+        name + " -" + delta + "%",
+        ConfigService.DecreaseVolumeButton
+      )
+
+      this.decreaseVolumeSwitch
+        .getCharacteristic(Characteristic.On)
+        .on(
+          CharacteristicEventTypes.SET,
+          this.adjustVolume.bind(this, logarithmic, -delta, delay)
+        )
+        .on(CharacteristicEventTypes.GET, this.showAlwaysOff.bind(this))
+    }
+
+    if (initialVolume != null) {
+      log.debug("Setting initial volume")
+      this.setVolume(logarithmic, initialVolume, () => {
+        /* do nothing */
+      })
+    } else if (this.cached) {
+      log.debug(
+        "Using cached volume without initial value and thus reading current system volume"
+      )
+      loudness.getVolume().then((homekitVolume) => {
+        this.currentVolume = homekitVolume
+      })
+    }
+    if (initiallyMuted != null) {
+      log.debug("Setting initial mute status")
+      this.setMuted(initiallyMuted, () => {
+        /* do nothing */
+      })
+    }
+
+    setTimeout(
```

Comment:
The purpose indeed is to initially update the services.  I thought it would be necessary.  If it works without the code block, then it can surely be removed.